Dockerfile

```diff
@@ -0,0 +1,44 @@
+FROM ubuntu:19.10 AS deps
+
+ENV DOTNET_SPARK_VERSION 0.10.0
```

Comment:
Is it possible to make this not dependent on a specific version in this file so that we don't have to update this every time we release a new version?

Reply:
I'm not sure how to do that. Is there a way to get the current version?